_posts/en/newsletters/2019-11-27-newsletter.md

```diff
@@ -0,0 +1,280 @@
+  then be copied into another application for signing (e.g. [HWI][topic
+  hwi]).  The GUI doesn't yet provide a special dialog for copying the
+  signed PSBT back in for broadcasting.
+
+- [Bitcoin Core #17290][] changes which coin selection algorithm is used
+  in cases where the user requests certain inputs be used or asks that
+  the fee be selected from the payment amounts.  These now use Bitcoin
+  Core's normal default algorithm of Branch and Bound (BnB).  BnB was
+  designed to minimize fees and maximize privacy by optimizing for the
+  creation of changeless transactions.
+
+- [C-Lightning #3264][] includes several mitigations for [LND
+  #3728][], a bug in the implementation of gossip queries.  This
+  change also adds two new command line parameters useful for testing
+  and debugging, `--hex` and `--features`.
+
+- [C-Lightning #3274][] causes `lightningd` to refuse to start if it
+  detects that `bitcoind` is now on a lower block height than it was the
+  last time `lightningd` was run.  If a lower height is seen while
+  `lightningd` is running, it will simply wait until a higher height is
+  seen.  Block heights can decrease during a block chain reorganization,
+  during a block chain reindex, or if the user runs certain commands
+  intended for developer testing. <!-- e.g.  invalidateblock --> It's
+  easier and safer for `lightningd` to wait for those situations to be
+  resolved by `bitcoind` than it is to try to work around the problems.
+  However, if the LN user really wants to use the truncated chain, they
+  can start `lightningd` with the `--rescan` parameter to reprocess the
+  block chain.  <!-- yes, that parameter is for lightningd (not
+  bitcoind); see test in this PR -->
+
+- [Eclair #1221][] adds a `networkstats` API that returns various
+  information about the LN network as observed from the local node,
+  including the number of known channels, number of known LN nodes, the
+  capacity of LN nodes (grouped into percentiles), and the fees nodes are
+  changing (also grouped into percentiles).
+
+- [LND #3739][] makes it possible to specify what node should be the
+  last hop on route before a payment is delivered to the receiver.
+  In conjunction with other still-pending work, such as [LND #3736][], this
+  will make it possible for a user to rebalance their channels using
+  built-in LND features (instead of requiring external tools, as is
+  currently the case).
+
+- [LND #3729][] makes it possible to generate invoices with millisatoshi
+  precision.  Previously, LND would not generate invoices with
+  sub-satoshi precision.
+
+- [LND #3499][] extends several RPCs, such as `listpayments` and
+  `trackpayments` to provide information about [multipath
+  payments][topic multipath payments], payments that can have multiple
+  parts that are sent over different routes.  These are not fully
+  supported by LND yet, but this merged PR makes it easier to add
```

Comment:
to add -> to adding

Reply:
I don't think so?  (Your corrections are usually so good that I'm doubting my sanity here.)

Reply:
I don't think so either!

Reply:
I read it over three times you're right. Direct quote from @jnewbery, "Jonas is open to adding grammatical errors to the newsletter."